src/synology_dsm/synology_dsm.py

```diff
@@ -8,6 +8,7 @@
 from requests import Session
 from requests.exceptions import RequestException
 
+from .api.audio_station import SynoAudioStation

```

Comment:
please also add handling for SynoAudioStation in `update()` and `reset()` method below (_compare with SynoSurveillanceStation or SynoSurveillanceStation_)

Reply:
Ty for all the comments, will add some tests as soon as I got some time

When I look at the the update / reset mechanism it's to provide some kind of cache ?
Since the AudioStation part I added is mainly to control / monitor realtime music. Does is make sense to cache a playlist status that can change every few seconds ?
If I add the `update()` / `reset()` part which info should actually be updated ? The remote player info, current playlist, the info from the `getInfo` method (got to add it 😄 ) ?

It makes me think more about if I' m correct to add it as an "AudioStation" class, or should I split it in different classes :
AudioStationInfo
AudioStationRemotePlayer
AudioStationRemotePlayerStatus
This design would be more similar to the `core` part I guess ?

I realize that I kinda went with a different approach than the rest of the lib, returning the objects on calls. And not storing them on the model itself